src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptions.java

```diff
@@ -31,6 +31,7 @@
         public boolean enableVignette = true;
         public boolean enableFog = true;
         public boolean enableClouds = true;
+        public boolean staticFov = true;
```

Comment:
This should be false by default since it's a cosmetic change to vanilla behavior, IMO.

Reply:
Yeah that's true, I'll change this over in a few.